imagetest/test_suites/storage_perf/startupscripts/install_fio.sh

```diff
@@ -0,0 +1,16 @@
+if [[ -f /usr/bin/apt ]]; then
```

Comment:
It may be better to do this with if, elif. In the odd case that more than one of these tools exists (it really shouldn't but let's say they do), one of the calls would fail. The only case that is like this is yum/dnf where there is a symlink to yum on dnf based systems and therefore you'll end up calling the install twice.

Reply:
done